data/releases/schedule.yaml

```
@@ -1,4 +1,16 @@
 schedules:
+- release: 1.27
+  releaseDate: 2024-04-11
+  maintenanceModeStartDate: 2024-04-28
+  endOfLifeDate: 2024-06-28
+  next:
+    release: 1.27.1
+    cherryPickDeadline: 2023-04-07
```

```suggestion
    cherryPickDeadline: 2023-04-15
```
Is this the correct date?

---

Yes, that's the correct date.